### ISSUE TITLE
Suggestion for rule

### ISSUE THREAD

**Issue by baarsrj** [languagetool-org/languagetool#7790]
Suggest a non breaking space between a number and it's unit.

**Comment by Discostu36** [comment]
Non-breaking spaces are not allowed or do not work in many contexts, especially in the web browser. So I guess this would only be a good idea for LibreOffice.

**Comment by Discostu36** [comment]
See also #1237

**Comment by baarsrj** [comment]
If it does not work in the web browser, that should be considered a bug there, and reported there as such.
